swarmrl/gyms/gym.py

```diff
@@ -30,7 +31,7 @@ class Gym:
     def __init__(
         self,
         rl_protocols: List[ActorCritic],
-        loss: Loss,
+        loss: Loss = ProximalPolicyLoss,
```

Comment:
What are the pros and cons of having PPO as the default. One one hand it is the better algorithm but on the other it is much more expensive.  Perhaps having gradient optimisation as the default is the simpler option. What do you think?

Reply:
also from a software standpoint: do you want the loss by default to be the Class PPO or an instance of PPO? In case of the latter, you need braces () to instantiate an object

Reply:
Ahh very good point. This should be an instance not a class.

Reply:
Yeah so let's put no default in the gym. 
Only put defaults in the losses (GAE and Gumble) 
And make it actual instances.

Reply:
There are two points here though, if you want this to be a default, it must be an instance. Otherwise the code won't run.

We can totally put the loss as default in the Gym I am completely for that as it simplifies everything for the user. I would however go with the faster and more basic gradient optimisation.

Reply:
Resolved in https://github.com/SwarmRL/SwarmRL/pull/9/commits/fc718a42587e05f369b612b618cfc985f4ff3b6e